### ISSUE TITLE
Postman runner doesn't sort collections or environments anymore

### ISSUE THREAD

**Issue by chrisron95** [a85/POSTMan-Chrome-Extension#4311]
## App Details:

```
Postman for Mac
Version 6.0.9
OS X 17.4.0 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:
Only since updating to 6
 2. Expected behaviour:
collections and environments should be sorted. Especially when working with many collections and environments, if there's no sorting, it's not always easy to find what you're looking for and it's very easy to miss the one you're looking for while scrolling through.
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
I think most of these logs have to do with recent runs not showing up, which is a separate issue already written up, so I'm not sure how useful these logs are.
```
Initializing the global config
runner.js:328755 Initializing settings
runner.js:266326 Booting waterline models
runner.js:266392 
Object
 "Successful ORM initialization"
runner.js:344471 
OperationalError
runner.html:1 Uncaught (in promise) 
OperationalError
__stackCleaned__
:
true
adapterMethodName
:
"find"
cause
:
Error: Malformed error from adapter: Should always be an Error instance, but instead, got: ``` [UnknownError: Maximum IPC message size exceeded.] ``` at file:///Applications/Postman.app/Contents/Resources/app/js/runner.js:29660:20
isOperational
:
true
message
:
"Malformed error from adapter: Should always be an Error instance, but instead, got:↵```↵[UnknownError: Maximum IPC message size exceeded.]↵```"
modelIdentity
:
"collectionrun"
name
:
"AdapterError"
raw
:
DexieError
stack
:
"AdapterError: Malformed error from adapter: Should always be an Error instance, but instead, got:↵```↵[UnknownError: Maximum IPC message size exceeded.]↵```↵    at file:///Applications/Postman.app/Contents/Resources/app/js/runner.js:29660:20↵From previous event:↵    at Deferred.toPromise (file:///Applications/Postman.app/Contents/Resources/app/js/runner.js:272245:19)↵    at Deferred.then (file:///Applications/Postman.app/Contents/Resources/app/js/runner.js:272106:22)"
__proto__
:
Error
```
 4. Screenshots (if applicable)
![screen shot 2018-03-08 at 3 01 10 pm](https://user-images.githubusercontent.com/16259311/37173726-5c0e8670-22e2-11e8-9624-bb9e40ba8122.png)

**Comment by sivcan** [comment]
Hi @chrisron95 - Thanks for reporting this. 
I apologise for the inconvenience caused.

This has been fixed on our end and you can expect the same to be reflected in the upcoming release.
I'll update this thread once it's out.

**Comment by serg1409** [comment]
See also issue #4272. Waiting for your fix. 
Thanks

**Comment by sivcan** [comment]
Hi @chrisron95 - The new update has been released (6.0.10) and has the fix for this issue.
Closing this.
